table_connector.py

```diff
@@ -44,9 +44,13 @@ def __get_latest_dir_in_dir(directory):
     return newest_path
 
 
-# Removes a leading one if it exists.
-def __strip_initial_1_in_phone(row):
+# Removes leading ones from phone numbers as well as any spaces or punctuation.
+def __standardize_phone_numbers(row):
     phone_or_email = row.phone_or_email
+    if '@' not in phone_or_email:
+        # Change unbreakable space characters into regular spaces.
+        phone_or_email = phone_or_email.replace(u'\xa0', u' ')
+        phone_or_email = re.sub('[()\- ]+', '', phone_or_email)
```

Comment:
Add an "r" before the string here to make it a raw string.  It works fine now, but if you were to add something after that slash which is special such as "n" or "t" it would break and you wouldn't know why.